ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -92,7 +92,7 @@ object Printers {
 
         case _ =>
           print('{'); indent(); println()
-          print(tree)
+          printOne(tree)
```

Comment:
This `printOne`, as well as many others, could just as well directly call the specialized `print` for `Tree`. There should be very few calls to `printOne` left, IMO.